integration-tests/test/ovmcontext.spec.ts

```diff
@@ -100,11 +76,15 @@ describe('OVM Context: Layer 2 EVM Context', () => {
   })
 
   it('should set correct OVM Context for `eth_call`', async () => {
-    const tip = await L2Provider.getBlockWithTransactions('latest')
-    const start = Math.max(0, tip.number - 5)
-
-    for (let i = start; i < tip.number; i++) {
-      const block = await L2Provider.getBlockWithTransactions(i)
+    for (let i = 0; i < numTxs; i++) {
```

Comment:
This test was previously relying on the fact that the test before it was being executed correctly. Now it's independent of the previous test.